prost-build/src/ast.rs

```diff
@@ -83,10 +85,24 @@ impl Comments {
                 buf.push_str("    ");
             }
             buf.push_str("///");
-            buf.push_str(line);
+            buf.push_str(&Self::sanitize_line(line));
             buf.push('\n');
         }
     }
+
+    /// Sanitizes the line for rustdoc by performing the following operations:
+    ///     - escape urls as <http://foo.com>
+    ///     - escape `[` & `]`
+    fn sanitize_line(line: &str) -> String {
+        lazy_static! {
```

Comment:
Not totally sure if we really need lazy static but don't think it hurts that much.